docs/sdks/python/agents.mdx

```diff
@@ -1,137 +1,348 @@
 ---
-title: AI Agents with LLMs and Skills
-sidebarTitle: Agents
+title: How to Use Agents
+sidebarTitle: AI Agents
 ---
 
-## Description
+Agents enable conversation with data, including structured and unstructured data connected to MindsDB.
 
-With MindsDB, you can create and deploy AI agents that comprise AI models and customizable skills such as knowledge bases and text-to-SQL.
+## Deploy Agents
 
-<p align="center">
-  <img src="/assets/agent_diagram.png" />
-</p>
+Here is the syntax for creating an agent:
 
-AI agents comprise of skills, such as text2sql and knowledge_base, and a conversational model.
-
-* Skills provide data resources to an agent, enabling it to answer questions about available data. Learn more about [skills here](/sdks/python/agents_skills). Learn more about [knowledge bases here](/sdks/python/agents_knowledge_bases).
-
-* A conversational model (like OpenAI) from LangChain utilizes [tools as skills](https://python.langchain.com/docs/how_to/#tools) to respond to user input. Users can customize these models with their own prompts to fit their use cases.
+```python
+# prerequisite: get an existing project
+project = server.get_project()
+print(project)
+
+# create an agent
+agent = project.agents.create(
+        'my_agent',
+        model={
+            'model_name': 'gpt-4o',
+            'provider': 'openai',
+            'api_key': 'sk-abc123',
+            'base_url': 'http://example.com',
+            'api_version': '2024-02-01'
+        },
+        data={
+            'knowledge_bases': ['project_name.kb_name', ...]
+            'tables': ['datasource_conn_name.table_name', ...]
+        },
+        prompt_template='describe data'
+)
+```
 
-## Syntax
+It creates an agent that uses the defined model and has access to the connected data. Here is how to list all available agents.
 
-### Creating an agent
+```python
+agents = project.agents.list()
+print(agents)
+```
 
-When creating an agent, you can use the default conversational model:
+Alternatively, you can get an existing agent:
```

Comment:
Should we move this to a new section (maybe 'Get Existing Agents') and move the parameters related to creating agents to take its place? I feel like there is a break there.